src/types/gecko-profile.js

```diff
@@ -167,6 +167,8 @@ export type GeckoProfile = {|
     stackwalk: number,
     toolkit: string,
     version: number,
+    appBuildID: string,
+    sourceURL: string,
```

Comment:
I think these fields need to be marked with question marks, because older profiles don't contain them and we haven't increased the profile version to require the presence of these fields.

Reply:
Somebody could go along and add code that does `profile.meta.sourceURL.startsWith("https://")` (just as an example), and flow would happily accept it. But at runtime, it would cause an error for old profiles because `profile.meta.sourceURL` would be undefined and not have a `startsWith` method.

If flow knows that the field can be undefined, it will enforce a check before the field can be used as a string.

Reply:
Got it, thanks!